KubeArmor/kvmAgent/kvmAgent.go

```diff
@@ -0,0 +1,120 @@
+package kvmAgent
+
+import (
+	"context"
+	"encoding/json"
+	"errors"
+	"io"
+	"log"
+	"os"
+
+	tp "github.com/kubearmor/KubeArmor/KubeArmor/types"
+	pb "github.com/kubearmor/KubeArmor/protobuf"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/metadata"
+)
+
+// Variables
+var (
+	client           pb.KVMClient
+	identity         string
+	grpcClientConn   grpc.ClientConn
+	UpdateHostPolicy func(tp.K8sKubeArmorHostPolicyEvent)
+)
+
+func getGrpcConnAddress() string {
+	return (os.Getenv("CLUSTER_IP") + ":" + os.Getenv("CLUSTER_PORT"))
```

Comment:
Best to use `net.JoinHostPort`. This will take care of IPv6 address handling as well.